packages/design-system-docs/src/pages/components/Table/_Table.docs.scss

```diff
@@ -24,8 +24,7 @@ Style guide: components.table.complex
 `<Table>`
 
 `Table` is a container component that contains `TableCaption`, `TableHead` and `TableBody` as children,
-as well as `TableRow`, `TableHeaderCell` and `TableDataCell` for the table content. While the usage of
-these components follows ordinary HTML table semantics, this React implementation includes additional
+as well as `TableRow` and `TableCell` for the table content. These components mostly follow ordinary HTML table semantics, but also include some additional 
 responsive features including horizontal scrolling and vertically stacked rows.
 
 @react-example Table.example.jsx
```

Comment:
Add responsive preview similar to what is found on other responsive pages https://design.cms.gov/utilities/font-size/

```suggestion
@react-example Table.example.jsx
@responsive
```

Reply:
Hm... I just wanted to point out we currently only use this in Patterns and on responsive utilities. We currently dont have any official standards on when to use this but this is probably fine...

Reply:
Yeah, I was thinking it would be easier for folks to know when the responsive classes kick in as the breakpoints are called out in the tabs 
![Screen Shot 2020-09-02 at 1 41 44 PM](https://user-images.githubusercontent.com/1449852/92017614-0bbf9f80-ed22-11ea-8c77-3c5e15c98b93.png)